### ISSUE TITLE
Unable to format constant value as a typescript literal

### ISSUE THREAD

**Issue by alansingfield** [frhagn/Typewriter#177]
Related to #101 
The Constant.Value property gives you the raw text of the constant. We need a method to quote and escape this (such as HttpUtility.JavascriptEncode() ) - BUT - there isn't an extension that allows you to use that method.

```
C#:   public const string SomeData = "Example";
.TST: $Constants(*)[public static readonly $name: $Type = $Value;]
.TS:  public static readonly someData: string = Example  // wrong!

```

There are one of two ways to solve this:
1. Add an extension to allow use of HttpUtility.JavascriptEncode (or equivalent)
2. Add a Constant.ValueLiteral property which gives you the value as a javascript literal. This should probably support dates, decimal numbers as well.

I cheated and just put quote marks around the value - all of the constants I'm using so far are very simple strings:
```
    string ValueLiteral(Constant c) {
        if(c.Type.Name == "string")
            return @"""" + c.Value + @"""";
        else
            return c.Value;
    }

```

**Comment by nicholashead** [comment]
Good catch - I had the same problem, and I couldn't get the .tst to recognize System.Web.HttpUtility as existing.
